components/clp-package-utils/clp_package_utils/scripts/start_clp.py

```diff
@@ -1105,16 +1068,16 @@ def start_garbage_collector(
     ]
     # fmt: on
 
-    necessary_env_vars = [
-        f"PYTHONPATH={clp_site_packages_dir}",
-        f"CLP_HOME={CONTAINER_CLP_HOME}",
-        f"CLP_LOGS_DIR={container_logs_dir}",
-        f"CLP_LOGGING_LEVEL={clp_config.garbage_collector.logging_level}",
-    ]
     necessary_mounts = [
         mounts.clp_home,
         mounts.logs_dir,
```

Comment:
Don't we need to add `mounts.generated_config_file` here?

Reply:
right we should. it was still functional without that mount because the logs directory was mounted, though we should really add it